components/testimonial/Testimonial.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './Testimonial.module.scss';
+
+function Testimonial() {
```

Comment:
lets change this to a named export arrow function 
`export const Testimonial = () => {
`

Then lets create an index.ts file in this directory and export the component from there
` export * from './Testimonial';`

Reply:
this will also require adding this line to the .eslintrc file at the bottom of the rules list
		`"import/export": 0`

Reply:
Done